docs/markdown/releasenotes.md

```diff
@@ -6,6 +6,12 @@
 * (IDETECT-2949) Fixed an issue where [solution_name] failed to properly parse Go module version names containing '-' characters.
 * (IDETECT-2959) Fixed an issue where [solution_name] would not fail when running `go mod why` fails.
 
+### Changed features
+* [solution_name] has moved the warning message when generating empty code locations from WARN to the DEBUG logging level.
```

Comment:
Change to read: "[solution_name] now classifies empty code location warning messages as the DEBUG logging level instead of the previous classification as the WARN logging level."